src/sage/topology/simplicial_set_examples.py

```diff
@@ -732,14 +732,14 @@ def HopfMap():
 
     Using the Hopf map to attach a cell::
 
-        sage: X = g.mapping_cone()
-        sage: CP2 = simplicial_sets.ComplexProjectiveSpace(2)
-        sage: X.homology() == CP2.homology()
+        sage: X = g.mapping_cone()                                                      # optional - sage.graphs
+        sage: CP2 = simplicial_sets.ComplexProjectiveSpace(2)                           # optional - sage.graphs
+        sage: X.homology() == CP2.homology()                                            # optional - sage.graphs
         True
```

Comment:
No `pyparsing`?

Reply:
No, only CP3 and CP4 need that. See `simplicial_set_examples.ComplexProjectiveSpace`

Reply:
OK.